force-app/main/default/quickActions/ServiceSchedule__c.Add_More_Participants.quickAction-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<QuickAction xmlns="http://soap.sforce.com/2006/04/metadata">
+    <height>250</height>
+    <label>Add More Participants</label>
+    <lightningComponent>addMoreParticipants</lightningComponent>
+    <optionsCreateFeedItem>false</optionsCreateFeedItem>
+    <type>LightningComponent</type>
+    <width>-100</width>
```

Comment:
Does this do anything since we have the width override?

Reply:
no, it's the default that comes when we pull. same on future session deleter.